From: Colin Ian King <colin.king@canonical.com>
Date: Fri, 29 Sep 2017 14:52:27 +0100
Subject: RDMA/hns: remove redundant assignment to variable j
Patch-mainline: v4.15-rc1
Git-commit: e930b4d8a52067c773d9cd6f6bdd7d4ddc99563f
References: bsc#1104427 FATE#326416

Variable j is being assigned to loop_j and then later being
assigned to a new value in for loops. The first initialization
is therefore redundant and can be removed.

Cleans up clang warning:
warning: Value stored to 'j' is never read

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/hns/hns_roce_mr.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/infiniband/hw/hns/hns_roce_mr.c
+++ b/drivers/infiniband/hw/hns/hns_roce_mr.c
@@ -272,7 +272,6 @@ static void hns_roce_loop_free(struct hn
 	mhop_num = hr_dev->caps.pbl_hop_num;
 
 	i = loop_i;
-	j = loop_j;
 	if (mhop_num == 3 && err_loop_index == 2) {
 		for (; i >= 0; i--) {
 			dma_free_coherent(dev, pbl_bt_sz, mr->pbl_bt_l1[i],
